tracer/test/Datadog.Trace.Tests/SpanContextPropagatorTests_ExtractHeaderTags.cs

```diff
@@ -0,0 +1,162 @@
+// <copyright file="SpanContextPropagatorTests_ExtractHeaderTags.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Linq;
+using System.Net;
+using Datadog.Trace.ExtensionMethods;
+using Datadog.Trace.Headers;
+using Xunit;
+
+namespace Datadog.Trace.Tests
+{
+    public class SpanContextPropagatorTests_ExtractHeaderTags
```

Comment:
These are tests for the `SpanContextPropagatorTests.ExtractHeaderTags()` overloads. I moved them from `HeadersCollectionTestHelpers.cs`.